packages/eslint-plugin/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.3.15](https://github.com/kriskowal/dotfiles/compare/@endo/eslint-plugin@0.3.14...@endo/eslint-plugin@0.3.15) (2021-12-08)
+
+**Note:** Version bump only for package @endo/eslint-plugin
```

Comment:
There's still something suspicious about this release process.  Why was it a version bump only if there were no changes except the version bump?

Reply:
And why is the named repository https://github.com/kriskowal/dotfiles ?

Reply:
That’s *really* weird because kriskowal/dotfiles is my personal home directory and should not appear here at all.

It might be that I should not attempt to do a release from a git workspace, since `.git` is a file instead of a directory. It might be reaching up to the `~/.git`.

Reply:
Yeah, added a reminder to MAINTAINERS and fixed the references from the prior release.